### PR TITLE
fix de new feature

### PR DESCRIPTION
**Pull request by hliebscher**

### CODE REVIEW

**Review by discip — CHANGES_REQUESTED** [review]
I'm pretty sure you didn't mean to translate **`Scarf Joint`** as **`Schaltnaht`**.

It should be something like **`Schäftung`**.

File: localization/i18n/de/OrcaSlicer_de.po [hunk 1]
```diff
@@ -11930,12 +11973,22 @@ msgid ""
...
 msgid "Scarf joint flow ratio"
-msgstr ""
+msgstr "Schaltnaht Flussverhältnis"
 
 msgid "This factor affects the amount of material for scarf joints."
-msgstr ""
+msgstr "Dieser Faktor beeinflusst die Menge an Material für Schaltnähte."
 
 msgid "Scarf start height"
 msgstr "Start Höhe des Schalnaht"
```
Comment (plexi32) [inline localization/i18n/de/OrcaSlicer_de.po:11994]: Besser wäre: Start-Höhe

File: localization/i18n/de/OrcaSlicer_de.po [hunk 2]
```diff
@@ -11930,12 +11973,22 @@ msgid ""
...
+"Geschwindigkeit auf der Grundlage der jeweiligen Geschwindigkeit der äußeren "
+"oder inneren Wand berechnet. Der Standardwert ist auf 100% eingestellt."
 
 msgid "Scarf joint flow ratio"
-msgstr ""
+msgstr "Schaltnaht Flussverhältnis"
 
 msgid "This factor affects the amount of material for scarf joints."
-msgstr ""
+msgstr "Dieser Faktor beeinflusst die Menge an Material für Schaltnähte."
```
Comment (plexi32) [inline localization/i18n/de/OrcaSlicer_de.po:11991]: Tippfehler: Schalnähte

File: localization/i18n/de/OrcaSlicer_de.po [hunk 3]
```diff
@@ -11930,12 +11973,22 @@ msgid ""
...
+"Geschwindigkeit der äußeren oder inneren Wände abweicht. Wenn die hier "
+"angegebene Geschwindigkeit höher ist als die Geschwindigkeit der äußeren oder "
+"inneren Wände, wird der Drucker auf die langsamere der beiden Geschwindigkeiten "
+"zurückgesetzt. Wenn sie als Prozentsatz angegeben wird (z.B. 80%), wird die "
+"Geschwindigkeit auf der Grundlage der jeweiligen Geschwindigkeit der äußeren "
+"oder inneren Wand berechnet. Der Standardwert ist auf 100% eingestellt."
 
 msgid "Scarf joint flow ratio"
-msgstr ""
+msgstr "Schaltnaht Flussverhältnis"
```
Comment (plexi32) [inline localization/i18n/de/OrcaSlicer_de.po:11988]: Schalnaht

File: localization/i18n/de/OrcaSlicer_de.po (hunk 1 above)
Comment (plexi32) [reply localization/i18n/de/OrcaSlicer_de.po:11994]: Schaltnaht kommt öfter fälschlicherweise vor.

**Review by plexi32 — CHANGES_REQUESTED** [review]

**Comment by discip** [comment]
@plexi32
In German there is no such term like **_Schalnaht_**.

As mentioned above, the name for this kind of junction is either **Schäftung** or **Schäftverbindung**.

**Comment by SoftFever** [comment]
Thanks all for the feedback 👍 
Unfortunately I can't give opinions on it.
Feel free to discuss it in Discord and submit PRs.

**Review by SoftFever — APPROVED** [review]
Thanks

**Pull request merged**